exercises/palindrome-products/description.md

```diff
@@ -21,8 +21,14 @@ palindromic product of factors within that range, along with all the factors in
 
 Given the range `[1, 9]` (both inclusive)...
 
-The smallest product is `1`. Its factors are `(1, 1)`.
-The largest product is `9`. Its factors are `(1, 9)`, and `(3, 3)`.
+And given the list of all possible products within this range:
+`[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 14, 16, 18, 15, 21, 24, 27, 20, 28, 32, 36, 25, 30, 35, 40, 45, 42, 48, 54, 49, 56, 63, 64, 72, 81]`
+
+The palindrome products are all single digit numbers (in this case):
+`[1, 2, 3, 4, 5, 6, 7, 8, 9]`
+
+The smallest palindrome product is `1`. Its factors are `(1, 1)`.
+The largest palindrome product is `9`. Its factors are `(1, 9)`, `(3, 3)`, and `(9, 1)`.
```

Comment:
[The tests](https://github.com/exercism/problem-specifications/blob/master/exercises/palindrome-products/canonical-data.json#L23) only expect (1,9) and (3,3) as factors.
(and always seem to expect factor pairs in (lowest, highest) order.

Reply:
Good point... I forgot I had to sort and uniq my results.   I'll udpdate.